test/kdf.js

```diff
@@ -61,14 +61,9 @@ test.cb('kdf.init(key, buffer) throws on bad input', (t) => {
   t.throws(() => kdf.init(123), TypeError)
   t.throws(() => kdf.init(true), TypeError)
   t.throws(() => kdf.init(() => undefined), TypeError)
-  t.throws(() => kdf.init(''), TypeError)
+  t.throws(() => kdf.init('a'), TypeError)
```

Comment:
@jwerle Is testing `'a'` valid? Should I check `''` instead and make sure that passes?